src/api/app/controllers/status_messages_controller.rb

```diff
@@ -0,0 +1,50 @@
+class StatusMessagesController < ApplicationController
```

Comment:
This controller and [Webui::StatusMessagesController](https://github.com/openSUSE/open-build-service/blob/42297257db7e8104ae758040d388be4ca0ac3cba/src/api/app/controllers/webui/status_messages_controller.rb) should be merged.

Reply:
Yes, like all the other API controllers we have. But that would be a second, or probably third step (second is to use pundit), and nothing can or want to do in this PR.